src/Components/Notifications/Notification.tsx

```diff
@@ -21,34 +23,40 @@ interface NotificationProps {
 }
 
 const Notification: React.FC<NotificationProps> = ({ notificationId }) => {
-  const { me } = useLazyLoadQuery<NotificationQuery>(notificationQuery, {
-    id: notificationId,
+  const { router } = useRouter()
+
+  const data = useLazyLoadQuery<NotificationQuery>(notificationQuery, {
+    internalID: notificationId,
   })
 
-  if (!me) {
-    return null
-  }
+  const notification = data.me?.notification
 
-  const { notification } = me
+  // Redirect user to the notifications targetHref if the notification type is not supported
+  useEffect(() => {
+    if (
+      !SUPPORTED_NOTIFICATION_TYPES.includes(
+        notification?.notificationType as string
+      )
+    ) {
+      router.push(notification?.targetHref as string)
+    }
+  }, [notification, router])
```

Comment:
Redirecting to the notifcation's `targetHref` in case the notification is not implemented.